distribution/src/resources/config-tool/templates/conf/security/cipher-text.properties.j2

```diff
@@ -0,0 +1,6 @@
+# By default, This file contains the secret alias names and the plain text passwords enclosed with '[]' brackets
+# In Production environments, It is recommend to replace these plain text password by the encrypted values. CipherTool can be used for it.
+
+{% for secret_name,secret_value in secrets.items()%}
+{{secret_name}}={{secret_value}}
+{% endfor %}
```

Comment:
Please add a new line , else this will create issue when creating installers.

Reply:
Fixed